README.org

```diff
@@ -58,6 +58,33 @@ configuration file:
 You can also decide to only activate beginend for some of its
 supported major modes (e.g., through ~beginend-dired-mode~).
 
+*** init.el example configuration
+
+By default, Beginend adds the ~" be"~ lighter to the modeline. You can use [[https://github.com/myrjola/diminish.el][diminish.el]] or [[https://savannah.nongnu.org/projects/delight][delight.el]] to turn this off for every minor mode Beginend defines by looping over the ~beginend-modes~ variable. Note that you will /also/ need to "turn off the lights" for ~beginend-global-mode~, which is not part of this list.
```

Comment:
it seems that Github's support for org-mode doesn't like `~" be"~`. I think that this would be enough:

> beginend adds `be` to the modeline